README.md

```diff
@@ -12,9 +12,9 @@ appropriately in the Makefile.
 
 To use, attach the SCL and SDA DDC lines (pins 15 and 16, respectively) of an
 HDMI breakout to the I2C lines on your AVR (which are PC5 and PC4, respectively,
-on the ATmega48/88/168/328), connect the HDMI port to your target device, then
-apply power to the device. It should enter BootROM USB DFU mode with no
-additional interaction needed.
+on the ATmega48/88/168/328), attach HDMI pin 17 to GND, and shunt HDMI pins 18
+and 19 together, then connect that to your target device and apply power to it.
```

Comment:
These are hotplug detect, right? Are they needed on every device? I don't remember having to do anything specific with them.

Reply:
The SoC uses hotplug detect as a sanity check to make sure a dongle is actually plugged in. It doesn't attempt i2c if there is nothing there.

Reply:
Which SoC? That might be new behavior, as I developed this with reference to an S905D3 BootROM dump, and that seems to poll I2C unconditionally without checking any GPIO first. I'll double check on real hardware, though, since maybe there's a check hidden somewhere and I got lucky by leaving it floating.

This is a good thing to mention regardless, but I'd like to be specific about whether it's needed for all chips or only for some.

Reply:
The one I was messing with is S905W4, so that one at the very least. And no, the W4 part is not a typo.